
WARNING: orio.module.loop.submodule.tile.transformation: tile size 2 must be greater than the stride value 2; tile index = iii
expected=2.448879
actual=2.500000
diff=0.000000
diff=1
expected=2.499818
actual=2.500000
diff=0.000000
diff=1
WARNING: orio.module.loop.submodule.tile.transformation: tile size 2048 must be greater than the stride value 2048; tile index = jjj
expected=2.500000
actual=2.500000
diff=0.000000
diff=1
expected=2.500000
actual=2.500000
diff=0.000000
diff=1
expected=2.499358
actual=2.500000
diff=0.000000
diff=1
expected=2.497753
actual=2.500000
diff=0.000000
diff=1
expected=2.500000
actual=2.500000
diff=0.000000
diff=1
expected=2.500000
actual=2.500000
diff=0.000000
diff=1
expected=4.995510
actual=2.500000
diff=2.000000
diff=0
[00;31mERROR: orio.main.tuner.ptest_driver: failed to execute the test code: "timeout --kill-after=30s --signal=9 20m  ./__orio_perftest9.exe  "
 --> CalledProcessError: Command 'timeout --kill-after=30s --signal=9 20m  ./__orio_perftest9.exe  ' returned non-zero exit status 1[00m
[00;31mERROR: orio.main.tuner.ptest_driver:  performance testing failed: "timeout --kill-after=30s --signal=9 20m  ./__orio_perftest9.exe  "[00m
expected=2.500000
actual=2.500000
diff=0.000000
diff=1
expected=2.500000
actual=2.500000
diff=0.000000
diff=1
expected=2.499920
actual=2.500000
diff=0.000000
diff=1
expected=2.497421
actual=2.500000
diff=0.000000
diff=1
expected=2.500000
actual=2.500000
diff=0.000000
diff=1
expected=2.493236
actual=2.500000
diff=0.000000
diff=1
WARNING: orio.module.loop.submodule.tile.transformation: tile size 512 must be greater than the stride value 512; tile index = iii
expected=2.497130
actual=2.500000
diff=0.000000
diff=1
expected=2.500000
actual=2.500000
diff=0.000000
diff=1
expected=2.485940
actual=2.500000
diff=0.000000
diff=1
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: Error in parse(text = x, keep.source = FALSE) : 
  <text>:2:0: unexpected end of input
1: ~
   ^

  warnings.warn(x, RRuntimeWarning)
Traceback (most recent call last):
  File "../../../orcc", line 27, in <module>
    orio.main.orio_main.start(sys.argv, orio.main.orio_main.C_CPP)
  File "/root/dlmt_spapt_experiments/orio/orio/main/orio_main.py", line 109, in start
    optimized_code_seq = odriver.optimizeCodeFrags(cfrags, True)
  File "/root/dlmt_spapt_experiments/orio/orio/main/opt_driver.py", line 45, in optimizeCodeFrags
    cur_seq = self.__optimizeCodeFrag(cf, perf_params)
  File "/root/dlmt_spapt_experiments/orio/orio/main/opt_driver.py", line 120, in __optimizeCodeFrag
    cfrag.cfrags)
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/tuner.py", line 152, in tune
    best_perf_params, best_perf_cost = search_eng.search()
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/search/search.py", line 132, in search
    best_coord,best_perf,search_time,speedup = self.searchBestCoord(startCoord)
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/search/dlmt/dlmt.py", line 878, in searchBestCoord
    best_point, used_points = self.dopt_anova()
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/search/dlmt/dlmt.py", line 771, in dopt_anova
    step_data = self.dopt_anova_step(budget, trials, i)
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/search/dlmt/dlmt.py", line 681, in dopt_anova_step
    output = self.opt_federov(encoded_full_model, trials, federov_search_space)
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/search/dlmt/dlmt.py", line 269, in opt_federov
    output = self.algdesign.optFederov(frml         = Formula(design_formula),
  File "/usr/lib/python2.7/dist-packages/rpy2/robjects/__init__.py", line 269, in __init__
    env = environment)
rpy2.rinterface.RRuntimeError: Error in parse(text = x, keep.source = FALSE) : 
  <text>:2:0: unexpected end of input
1: ~
   ^

